React/Base/RCTRootView.m

```diff
@@ -173,27 +173,34 @@ - (CGSize)sizeThatFits:(CGSize)size
   return fitSize;
 }
 
-- (void)layoutSubviews
```

Comment:
Shouldn't the iOS `-(void)layoutSubviews` be in the #else otherwise iOS will regress.

Reply:
Never mind, I see it now.